docs/cli.rst

```diff
@@ -69,6 +69,30 @@ See :doc:`data-process` for detail.
    Reference value for date-time substitutions.
    See also :envvar:`YP_TIME_REF_VALUE`.
 
+yp-preprocesor
+--------------
+
+Usage:
+
+.. code-block:: bash
+
+   yp-preprocesor [options] -o output-file-name input-file-name
+
+See :doc:`data-preprocessor` for detail.
+
+.. program:: yp-preprocessor
+
+.. option:: file-names
```

Comment:
Shouls this be:
` .. option:: --input_file=FILENAME, -i FILENAME `

Reply:
Not changed see above comments